server/controllers/doppler.controller.spec.js

```diff
@@ -0,0 +1,57 @@
+const chai = require('chai');
+const sinonChai = require('sinon-chai');
+const sinon = require('sinon');
+const DopplerController = require('./doppler.controller');
```

Comment:
const start with uppercase

Reply:
It's actually a class so it has to be in Uppercase 👍